src/math/lp/nla_grobner.cpp

```diff
@@ -498,9 +498,8 @@ namespace nla {
     }
 
     void grobner::prepare_rows_and_active_vars() {
-        m_rows.clear();
-        m_rows.resize(m_lar_solver.row_count());
-        c().clear_and_resize_active_var_set();
+        m_rows.reset();
+        c().clear_active_var_set();
```

Comment:
why remove m_rows.resize?

Reply:
m_rows is an indexed_uint_set. In the latter reset() is equivalent to clear() and there is no resize().